golem-worker-executor-base/src/storage/blob/mod.rs

```diff
@@ -155,9 +331,14 @@ pub enum BlobStorageNamespace {
         account_id: AccountId,
         worker_id: WorkerId,
     },
+    CompressedOplog {
+        account_id: AccountId,
+        component_id: ComponentId,
```

Comment:
is there reason why here is `component_id` and not `worker_id`?

Reply:
This way it was easy to implement the equivalent of redis SCAN without changing / extending the blob storage interface.